src/webui/www/private/views/search.html

```diff
@@ -921,7 +948,7 @@
                     offset: state.rowId
                 },
                 onFailure: function(response) {
-                    if (response.status === 400) {
+                    if (response.status === 400 || response.status === 404) {
```

Comment:
Will update. Small style changes like this are best enforced with eslint rules so that you don't have to spend time searching for them. When paired with a tool like prettier for auto-formatting, you never have to think about style again.

Reply:
I did looked for such option in eslint in the past but didn't found one. I did a second try after seeing your comment and succeeded this time. It took me a while to figure it out. Will submit a PR later.